notebook2/deployment.yaml.in

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: notebook2
+  labels:
+    app: notebook2
+    hail.is/sha: "@sha@"
+spec:
+  selector:
+    matchLabels:
+      app: notebook2
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: notebook2
+        hail.is/sha: "@sha@"
+    spec:
+      serviceAccountName: notebook2
+      containers:
+      - name: notebook2
+        image: @image@
+        ports:
+        - containerPort: 5000
+        volumeMounts:
+          - mountPath: /notebook-secrets
+            readOnly: true
+            name: notebook-secrets
+      volumes:
+        - name: notebook-secrets
+          secret:
+            secretName: notebook2-secrets
```

Comment:
It's probably failing to start because it can't mount notebook2-secrets.  Do you just want to reuse notebook-secrets?

Reply:
Yeah, I thought I had corrected that. Shoot.

Reply:
Addressed